docs/Data_Portal/Users_Guide/mutation_frequency.md

```diff
@@ -168,6 +168,6 @@ A set of frequently-used properties are available to filter genes and mutations
 * __Consequence Type:__  Consequence type of this variation; [sequence ontology](http://www.sequenceontology.org/) terms
 * __Type:__ A general classification of the mutation
 
-### Saving a Gene or Mutation Set
+## Saving a Gene or Mutation Set
```

Comment:
just thought this should be higher in the hierarchy cos it wasn't specific to just the facets filtering.